### PR TITLE
Display correct string on FAB [6525]

### LINKED ISSUE #6525: fab showing "Resume" instead of "Start" after marking all chapters as unread
### Steps to reproduce

1. Mark all chapters as read
2. Then mark all chapters as unread

https://user-images.githubusercontent.com/87689027/151727578-e52296a9-1101-46b7-9d32-a9eb5128dafb.mp4

### Expected behavior

fab shud display "Start"

### Actual behavior

fab shows as "Resume" instead

### Crash logs

_No response_

### Tachiyomi version

r4121

### Android version

11

### Device

Redmi Note 10

### Other details

Also happens when marking one chapter as read then marking it again as unread

https://user-images.githubusercontent.com/87689027/151727753-5b7dd1ca-13c3-4539-9cb0-8c62d10764b8.mp4

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by midyHamdoun**
Closes #6525.

Added an else clause to the `updateFabVisibility()`.

### CODE REVIEW

File: app/src/main/java/eu/kanade/tachiyomi/ui/manga/MangaController.kt [hunk 1]
```diff
@@ -382,6 +382,8 @@ class MangaController :
         val fab = actionFab ?: return
         if (adapter.items.any { it.read }) {
```
Comment (arkon) [inline app/src/main/java/eu/kanade/tachiyomi/ui/manga/MangaController.kt:383]: Can you move the show/hide calls into these blocks too?

Comment (FourTOne5) [reply app/src/main/java/eu/kanade/tachiyomi/ui/manga/MangaController.kt:383]: this might be the wrong line arkon

Comment (arkon) [reply app/src/main/java/eu/kanade/tachiyomi/ui/manga/MangaController.kt:383]: I actually sort of misread them (`all` and `any` look very similar sometimes...)

**Comment by arkon** [comment]
Ended up just keeping the original change -- 319e4360c8c9c27759be81a9e3836d2e50ece9a5

**Pull request closed without merging**